src/Common/FitOnFhir.Common/FitOnFhir.Common.csproj

```diff
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <CodeAnalysisRuleSet>..\..\..\CustomAnalysisRules.ruleset</CodeAnalysisRuleSet>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>	
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
```

Comment:
So, we need to make a decision here. We should either enable Nullable for all projects in this repository or remove it from all projects. It will impact the way code is written across the entire project and we should be consistent. Keep in mind, enabling it across the entire project is quite a big refactor.

We should also do the same for ImplicitUsings. I think we should add this to all the projects because the refactor is not that big (only a few files) and it is pretty useful.